source/introduction.txt

```diff
@@ -133,7 +133,7 @@ third parties to develop storage engines for MongoDB.
    :hidden:
 
    /tutorial/getting-started
-   /tutorial/atlas-free-tier-setup
+   Create an Atlas Free Tier <https://docs.atlas.mongodb.com/getting-started/>
```

Comment:
Just wondering.  Do we even want this in the toc? or should we just get rid of it all together. I think it made more sense around 3.4 when Atlas was starting out.   We still benefit from better SEO but not sure,

Reply:
@gilberto-mdb What do you think?